modules/hcp-dr-oadp-restore-auto.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-disaster-recovery-oadp-auto.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-dr-oadp-restore-auto_{context}"]
+= Restoring a hosted cluster by using {oadp-short}
+
+You can restore the hosted cluster by creating the `Restore` custom resource (CR).
+
+* If you are using an _in-place_ update, InfraEnv does not need spare nodes. You need to re-provision the worker nodes from the new management cluster.
+* If you are using a _replace_ update, you need some spare nodes for InfraEnv to deploy the worker nodes.
```

Comment:
Not sure you need the italics here. You are not defining the term, and in other places where there were different paths for different objects, you didn't italicize. Will typical users know which update they are doing?
Also, do we need to state what InfraEnv is?
```suggestion
* If you are using an in-place update, InfraEnv does not need spare nodes. You need to re-provision the worker nodes from the new management cluster.
* If you are using a replace update, you need some spare nodes for InfraEnv to deploy the worker nodes.
```